app/controllers/catalog_controller.rb

```diff
@@ -116,14 +116,37 @@ def self.modified_field
       title_name = solr_name("title", :stored_searchable)
       field.solr_parameters = {
         # qf: "#{all_names} file_format_tesim all_text_timv",
-        qf: [ solr_name("title", :stored_searchable),
-              solr_name("description", :stored_searchable),
-              solr_name("subject", :stored_searchable),
-              solr_name("creator", :stored_searchable),
+        qf: [ solr_name("affiliation", :stored_searchable),
+              solr_name("alternative", :stored_searchable),
+              solr_name("ark", :stored_searchable),
+              solr_name("available", :stored_searchable),
+              solr_name("based_near", :stored_searchable),
```

Comment:
@seanaery I think you may want `based_near_label` here instead of `based_near`, unless you're specifically interested in the geoname URI for the location.  E.g., from a solr record on rdr-dev ...
```
        "based_near_tesim":["http://sws.geonames.org/4464368/"],
        "based_near_label_tesim":["Durham"],
```

Reply:
@coblej Ah, yes--thank you! Good catch.